masonry/src/widgets/checkbox.rs

```diff
@@ -277,6 +273,36 @@ impl Widget for Checkbox {
             };
             scene.stroke(&style, Affine::IDENTITY, brush.color, None, &path);
         }
+        // Paint focus indicator around the entire widget (box + label)
+        if ctx.is_focus_target() || is_hovered {
+            let widget_size = ctx.size();
+
+            let focus_rect = Rect::new(0.0, 0.0, widget_size.width, widget_size.height);
+
+            let focus_rect = focus_rect.inflate(2.0, 2.0);
```

Comment:
I do slightly worry that if there's a "tight" parent around this widget, that this would overlap in bad ways. But I don't think we need to change anything in this PR